pkg/monitoring/metrics/ovirt-populator/populator-metrics.go

```diff
@@ -0,0 +1,34 @@
+package ovirtpopulator
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	ioprometheusclient "github.com/prometheus/client_model/go"
+)
+
+var (
+	populatorMetrics = []operatormetrics.Metric{
+		populatorProgress,
+	}
+
+	populatorProgress = operatormetrics.NewCounterVec(
+		operatormetrics.MetricOpts{
+			Name: "ovirt_progress",
```

Comment:
Same. The metric requires kubevirt_cdi_ prefix

Reply:
Also missing _total suffix, since it's a counter.

Reply:
@bennyz wdyt?